### PR TITLE
Kibana+Elasticsearch in docker containers template

### PR DESCRIPTION
**Pull request by ymartel06**
This template allows to deploy an Ubuntu Server 15.04 VM with Docker and starts a Kibana container listening on port 5601 which uses Elasticsearch database.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ymartel06__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@ymartel06__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: docker-kibana-elasticsearch/metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a Kiabana dashboard with Docker",
```
Comment (ahmetalpbalkan) [inline docker-kibana-elasticsearch/metadata.json]: typo

**Comment by singhkay** [comment]
lgtm, merged

**Comment by ymartel06** [comment]
Thanks!

**Pull request merged**